### PR TITLE
126 update quizes on frontend

### LINKED ISSUE #126: Update quiz'es on frontend

### PR DESCRIPTION
**Pull request by perkynades**
Closes #126

### CODE REVIEW

**Review by EmelijonJ — APPROVED** [review]
Very nice

File: web/app/context/QuizContext.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ export interface IAlternative {
     id?: number
     text: string,
     isCorrect?: boolean
+    id?: number
```
Comment (ivarhe) [inline web/app/context/QuizContext.tsx:8]:
```suggestion
```
Duplicate id?

File: web/app/routes/edit/$quizId.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    const [quizInfo, setQuizInfo] = useState<IQuizInfo>({
+        name: quiz.name,
+        description: quiz.description
+    })
+
+    const [questions, setQuestions] = useState<IQuestion[]>(quiz.questions || [])
+    
+    const onUpdateQuiz = async () => {
+        // Update quiz info
+        // @ts-ignore
```
Comment (ivarhe) [inline web/app/routes/edit/$quizId.tsx:31]: Move ts-ignore?

File: web/app/routes/edit/$quizId.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                        },
+                        method: 'PATCH'
+                    })
+                }
+            })
+
+
+            // Check if a question has been deleted
+            quiz.questions?.filter(x => !questions.includes(x)).map(async (question: IQuestion) => {
+            // @ts-ignore
```
Comment (ivarhe) [inline web/app/routes/edit/$quizId.tsx:65]: Remove?

**Review by ivarhe — CHANGES_REQUESTED** [review]

**Review by Henrhh — APPROVED** [review]

**Pull request merged**